contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/SearchOrBrowseWindow.vue

```diff
@@ -131,17 +135,15 @@
               last: this.$route.query.last || this.$route.path,
```

Comment:
Would it be possible to use

```this.$route.query.last || this.$route.name```

instead of

```this.$route.query.last || this.$route.path``` 

to keep our redirect strategy consistent? We use page names for navigation back everywhere, and I haven't seen a use-case where it would cause issues so far. In case there is some additional information saved in the path, we agreed to use more explicit routes. If this is some edge case, I think it'd be fine to merge and iterate later, though so far it seems to me that it should be fine.

Reply:
Note: there's an issue where some params are available on one route but not the other, which means we can't rely on just the name to go back. Going to leave as is for now and iterate on it later